lib/l10n-en_GB.js

```diff
@@ -182,6 +182,9 @@ for extended information about the publication rules.'
 ,   "style.script.not-found":   "A link to the script <code>fixup.js</code> is required (eg, <code>&lt;script src=\"//www.w3.org/scripts/TR/2016/fixup.js\"&gt;&lt;/script&gt;</code>)."
     // style/back-to-top
 ,   "style.back-to-top.not-found":  'It is recommended to include this <em>back-to-top</em> hyperlink at the end of <code>&lt;body&gt;</code>: <code>&lt;p role="navigation" id="back-to-top"&gt;&lt;a href="#toc"&gt;&lt;abbr title="Back to top"&gt;&uarr;&lt;/abbr&gt;&lt;/a&gt;&lt;/p&gt;</code>.'
+    // style/body-toc-sidebar
+,   "style.body-toc-sidebar.class-found": 'The class "toc-sidebar" should be added by fixup.js.'
+,   "style.body-toc-sidebar.selector-fail": 'Selector "body" failed.'
```

Comment:
Maybe better to format tag names, classes, etc with HTML, as we're doing with other rules?

    The class <code>toc-sidebar</code> should be added by <code>fixup.js</code>.

and:

    Selector <code>body</code> failed.

(I think the output looks better and is easier to understand.)